vars/common-vars.yml

```diff
@@ -64,6 +66,44 @@ suse_airship_components_image_tag: "{%- if suse_airship_build_local_images or su
 
 socok8s_site_name: "soc"
 
+# suffix used in FQDN for ingress (i.e. "publicly facing" access point)
+# FQDN -- > {component_api_name}.{namespace}.{cluster_domain_suffix}
+cluster_domain_suffix: "svc.dustycloud.local"
+
+# Only public endpoints are enabled and disabled from TLS perspective.
+# When TLS is enabled for an endpoint, corresponding TLS (tls.ca, tls.crt,
+# tls.key) attributes are made available for TLS secret and component
+# specific helm charts refer that secret in its ingress spec definition.
+
+# Flag to disable/enable TLS for ucp namespace public endpoints
+disable_tls_ucp: True
```

Comment:
why disable tls on ucp by default? It is exposed on vip ingress, so it is expected to be tls enabled.

Reply:
I'd like to see the default be TLS enabled unless there is a good reason not to. "Secure by default" is getting to be a common expectation, and the fewer things customers have to tweak manually to achieve strong security the better our product (or tech preview) looks.

Reply:
I got the answer form Arun. There are a few issues in UCP that blocks us from turning on TLS for UCP services, mainly around the client side cert is not installed correctly on cli side and between internal services. @bryanstephenson I saw Dirk had discussed with you that TLS is not a blocker for TP. In that case, are you okay if we deliver TP with TLS only on openstack service public end points, but not on Airship UCP services?

@kantarun I'd suggest to remove the ucp tls flag since it is not supported yet.

Reply:
Added comment here ..highlighting that this flag is not supported and so don't change.

Reply:
My understanding is that this is a large change very late in the game, and I think security is not the primary purpose of the tech preview (but demonstrating a use case). I totally agree that we should default to a secure configuration for Cloud 10.

Reply:
@dirkmueller I agree with what you said. Can you clarify that you are recommending this not to be merged as tech preview v1 entirely, or change the default to tls disabled for both openstank and airship ucp endpoints?

Reply:
we can merge it if its reviewed and not breaking the TP1. The default should not be changed at this point in time because it will invalidate existing testing and make CI getting green harder. if there is risk in merging the patch then I'm okay with postponing it after TP1.